common/dao-api/src/main/java/org/thingsboard/server/dao/user/UserService.java

```diff
@@ -30,34 +30,34 @@
 import java.util.List;
 
 public interface UserService extends EntityDaoService {
-	
```

Comment:
ty for formatting this. for future I suggest to format only whitespace but not the order of the methods like IDE likely do. It is impossible for me from UI to check what really added or removed